src/sentry/api/endpoints/prompts_activity.py

```diff
@@ -27,6 +29,27 @@ def validate_feature(self, value):
         return value
 
 
+class PromptsActivitiesEndpoint(Endpoint):
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request):
```

Comment:
@Neo-Zhixing just noticed this is a `post` request. Would be nice if we could do this as `get` request. Could this endpoint take a list of features and list of fields? I don't think we need to map `feature` too `field` (we just pass all the fields we need for each feature and pick the ones we need based on `required_fields`)

Reply:
I feel like if this is what we're going to do, we could totally reuse and modify the old end point so that it accepts multiple features.

Reply:
@Neo-Zhixing The problem with that approach is that we'll have two different return types. I'm ok with it but make sure you update the `PromptResponse` type to reflect that it could be an array in your PR.